Classes/Job/JobManager.php

```diff
@@ -101,4 +129,66 @@ public function peek($queueName, $limit = 1)
             return $job;
         }, $messages);
     }
-}
+
```

Comment:
Are these signals part of the public API? If so, we should mark them as API and add some quick doc for the arguments